content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -148,27 +148,27 @@ the version.
 ## Webhook conversion
 
 {{< note >}}
-Webhook conversion is intorduced in kubernetes 1.13 as an alpha feature. To use this feature, the
-`CustomResourceWebhookConversion` feature get should be set. Please refer to [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) documentation for more information.
+Webhook conversion is introduced in kubernetes 1.13 as an alpha feature. To use it, the
+`CustomResourceWebhookConversion` feature should be enabled. Please refer to the [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) documentation for more information.
 {{< /note >}}
 
 The above example has a None conversion between versions which only sets the `apiVersion` field
-on conversion and does not change the rest of the object. apiserver also supports webhook conversion that
-calls an external service in case of a conversion is required, e.g. when:
+on conversion and does not change the rest of the object. API server also supports webhook conversions that
+calls an external service in case of a conversion is required, For example when:
```

Comment:
The API server also supports webhook conversions that call an external service in case a conversion is required. For example when: